_pages/clients/python.md

```diff
@@ -64,206 +60,164 @@ client = Algorithmia.client("YOUR_API_KEY", "https://mylocalendpoint")
+{% highlight python %}
+# Download the file
+if client.file(response.result).exists() is True:
+    localfile = client.file(response.result).getFile()
+{% endhighlight %}
 
-For more information on the methods available using the Data API in Python check out the [Data API](http://docs.algorithmia.com/?python#data-api-specification) documentation or the [Python Client Docs](https://github.com/algorithmiaio/algorithmia-python).
+This copies the file from your data collection and saves it as a file on your local machine, storing the filename in the variable `localfile`.
 
-For convenience, here is the whole script available to run:
+Alternately, if you just need the binary content of the file to be stored in a variable, you can retrieve the remote file's content without saving the actual file:
 
 {% highlight python %}
-import Algorithmia
-from Algorithmia.acl import ReadAcl, AclType
-
-apiKey = "YOUR_API_KEY"
-# Create the Algorithmia client
-client = Algorithmia.client(apiKey)
+# Download contents of file as a string
+if client.file(response.result).exists() is True:
+    output = client.file(response.result).getBytes()
+{% endhighlight %}
 
-# Set your Data URI
-nlp_directory = client.dir("data://YOUR_USERNAME/nlp_directory")
-# Create your data collection if it does not exist
-if nlp_directory.exists() is False:
-    nlp_directory.create()
+This will get the image as binary data, saving it to the variable `input`, which might be useful when writing algorithms that are part of an image processing pipeline. For more image-manipulation tutorials, see the [Computer Vision Recipes]({{site.baseurl}}/tutorials/recipes/#computer-vision). 
 
-# Create the acl object and check if it's the .my_algos default setting
-acl = nlp_directory.get_permissions()  # Acl object
-acl.read_acl == AclType.my_algos  # True
+If the file was text (an image, etc), you could instead use the function `.getString()` to retrieve the file's content as a string. For more methods on how to get a file using the Data API from a data collection go to the [API Specification](http://docs.algorithmia.com/#getting-a-file).
 
-# Update permissions to private
-nlp_directory.update_permissions(ReadAcl.private)
-nlp_directory.get_permissions().read_acl == AclType.private # True
 
-text_file = "data://YOUR_USERNAME/nlp_directory/jack_london.txt"
+Additional Functionality
+------------------------
 
-# Upload local file
-if client.file(text_file).exists() is False:
-    client.file(text_file).putFile("/your_local_path_to_file/jack_london.txt")
+In addition to the functionality covered in this guide, the Python Client Library provides a complete interface to the Algorithmia platform, including managing Algorithms (and their versions and builds), organizations, users, source control, and the additional functionality of the Data API. You can find all the details, including Python examples, in the API Docs.
```

Comment:
Add links for API Docs, Data API, organizations, users, and source control

Reply:
Where there were relevant Dev Center links, I added those, otherwise I used API Docs links. Would all API Docs links be preferred?

Reply:
I just mean in this specific section to add https://algorithmia.com/developers/platform/organizations to "organizations", "source control" https://algorithmia.com/developers/algorithm-development/source-code-management, "API docs" https://algorithmia.com/developers/api/#introduction, "managing algorithms" https://algorithmia.com/developers/algorithm-development/algorithm-management more for a CTA then anything else - also let's remove "users" from that list because it's not really a page or section

Reply:
This was intended to reference the sections from the API Docs to provide a complete list of what you can do with the client, rather than links to more Dev Center pages. Is that not needed?

Reply:
Removed the API docs-specific links and replaced them with dev center links.